libs/ledger-live-common/src/families/tezos/signOperation.ts

```diff
@@ -43,7 +43,7 @@ export const signOperation = ({
         // disable the broadcast because we want to do it in a second phase (broadcast hook)
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-ignore
-        tezos.contract.context.injector.inject = async () => "";
+        tezos.contract.context.injector.inject = async () => "op4WsnE6gvDPFFzbXtsX1wLCsuAAbkA8JhXKApxvEYmaEd3fpNC"
```

Comment:
I don't really like this way to fix this... But couldn't find a quick win to fix it :/